Community Toolbox/objects/ui_DemoButton/Create_0.gml

```diff
@@ -0,0 +1,10 @@
+// Inherit the parent event
+event_inherited();
+
+on_click = method({ target_room, category }, function() {
```

Comment:
I learn so many neat GML tricks from reading your code. My way of making pseudo-closures is so much more obnoxiously verbose than this